Makefile

```diff
@@ -82,14 +82,20 @@ licenses: download ## Verifies dependency licenses
 verify: ## Verify code. Includes codegen, docgen, dependencies, linting, formatting, etc
 	go mod tidy
 	go generate ./...
-	hack/validation/kubelet.sh
-	hack/validation/taint.sh
-	hack/validation/requirements.sh
-	hack/validation/labels.sh
-	hack/validation/resources.sh
-	hack/validation/status.sh
-	cp -r pkg/apis/crds kwok/charts
-	hack/validation/kwok-requirements.sh
+	hack/validation/v1/kubelet.sh
+	hack/validation/v1/taint.sh
```

Comment:
Is there a way to parameterize these script files so that you pass in the apiversion to them and it just figures out which path that it should apply the change to?

Reply:
I'd imagine that the injected logic is going to look _extremely_ similar between the two versions

Reply:
Generally, if the only thing that's different is the prefix of each label's API version, then it'd be cool to just easily infer it from the folder that you're modifying.

Reply:
Planning on taking this up in a separate PR